README.md

```diff
@@ -419,6 +419,26 @@ $classification = $open_ai->classification([
 ]);
 ```
 
+## Chat
+
+Given a chat conversation, the model will return a chat completion response.
+
+ ```php
+$complete = $open_ai->chat([
+    'model' => 'gpt-3.5-turbo',
+    'messages' => [
+        [
+            "role" => "user", 
```

Comment:
Maybe educate the user that `role` can be either “system”, “user”, or “assistant”?

Reply:
I'll make micro-changes after merging for readme, thx.